### ISSUE TITLE
Rule react-a11y-input-elements incorrectly requiring value for file input

### ISSUE THREAD

**Issue by HiroshiOHSUGA** [Microsoft/tslint-microsoft-contrib#841]
### Bug Report

-   **`tslint-microsoft-contrib` version**: 6.1.0 
-   **TSLint version**: 5.13.1
-   **TypeScript version**: 3.3.3333
-   **Running TSLint via**: CLI

#### TypeScript code being linted

```tsx
import * as React from "react";

export const X: React.ReactNode = (
  <input
    type="file"
    accept="image/*"
    placeholder="this is file input"
  />
);
```

with `tslint.json` configuration:

```json
defaultSeverity: error
extends:
  - tslint-microsoft-contrib
  - tslint-config-prettier
jsRules:
rules:
```

#### Actual behavior
`ERROR: (react-a11y-input-elements) /....../X.tsx[4, 5]: Input elements must include default, place-holding characters if empty`

#### Expected behavior
no error

or

What value should  I set to value attribute ?

**Comment by JoshuaKGoldberg** [comment]
Perhaps @jcnsilva  or @lizzzp1 might have input? 😊

**Comment by lizzzp1** [comment]
https://github.com/Microsoft/tslint-microsoft-contrib/pull/754#discussion_r244640386
Seems like the error isn't specific enough, but should probably not require a value from certain inputs.

**Comment by JoshuaKGoldberg** [comment]
Oh is this issue a duplicate of #749 then?

@HiroshiOHSUGA could you try with v6.1.0 of the tslint-microsoft-contrib package?

**Comment by IllusionMH** [comment]
Actually looks like problem in logic: as I understand #749 and #754  - this test case shouldn't contain erros: [ReactA11yInputElementsRuleTests.ts#L34-L62](https://github.com/Microsoft/tslint-microsoft-contrib/blob/master/src/tests/ReactA11yInputElementsRuleTests.ts#L34-L62) after fix applied

**Comment by HiroshiOHSUGA** [comment]
Any updates on this issue?
